docs/supplychain-sbom.md

```diff
@@ -115,17 +115,22 @@ The color-coding is used in this document to help illustrate different SBOM acti
 > [!NOTE]
 > FIXME – Check if this is sane.
 
+<!--
+Note from Josh
+I'm not entirely convinced an assembler should exist as a section. It might make more sense to just refer to this as an author then break it out someday if that makes more sense.
+-->
+
 * SBOM Assemblers collect, assemble, update, or annotate SBOM metadata — _They make sure the metadata and related artifacts are **Current**_. (CPANSec-2024)
     * This role is very similar to SBOM Author roles, but while an SBOM Author mainly concerns themselves with the creation of authoritative meta fields, the SBOM Assembler ensures they are complete and correct.
 * They may for example collect SBOMs throughout build dependency resolution, and assemble (merge), translate (transform), to produce SBOMs for analysis or audit purposes. (NTIA-2021, "Transform" category, paraphrased)
 
 | Do | Field name                             | Required   | Data type    | CycloneDX 1.6                                                     | SPDX | Required by             |
 | -- | :------------------------------------- | :--------- | :----------- | ----------------------------------------------------------------- | ---- | ----------------------- |
 | 🟨 | SBOM Type                              | No         |              |                                                                   |      |                         |
-| 🟥 | SBOM Author                            | Yes        | Text         | bom.metadata.author                                               |      | NTIA-SBOM, DE-TR.5.2.1  |
-| 🟥 | SBOM Creation Time-stamp               | Yes        | DateTime     | bom.metadata.timestamp                                            |      | NTIA-SBOM, DE-TR.5.2.1  |
-| 🟨 | SBOM Generation Tool                   | No         | List         | bom.metadata.tools[]                                              |      |                         |
-| 🟥 | SBOM Serial Number                     | Yes        | UUID         | bom.metadata.serialNumber                                         |      |                         |
+| 🟥 | SBOM Author                            | Yes        | Text         | bom.metadata.author                                               | creationInfo.creators[]  | NTIA-SBOM, DE-TR.5.2.1  |
+| 🟥 | SBOM Creation Time-stamp               | Yes        | DateTime     | bom.metadata.timestamp                                            | creationInfo.created     | NTIA-SBOM, DE-TR.5.2.1  |
+| 🟨 | SBOM Generation Tool                   | No         | List         | bom.metadata.tools[]                                              | creationInfo.creators[] |                         |
```

Comment:
How does one distinguish the person/organization updating the SBOM document ("SBOM Author") from the tool ("SBOM Generation Tool") being used for doing the updating?

You seem to use the same SPDX array for both `creationInfo.creators[]`.

Reply:
The creator field is freeform, so it can be anything or nothing :)

https://spdx.github.io/spdx-spec/v2.3-RC1/document-creation-information/#610-creator-comment-field

For example if I look at a SPDX SBOM generated by Syft, I see
```
  "creators": [
   "Organization: Anchore, Inc",
   "Tool: syft-0.65.0"
  ],
```